src/Common/tests/System/AssertExtensions.cs

```diff
@@ -50,5 +52,31 @@ public static T Throws<T>(string paramName, Func<object> testCode)
 
             return exception;
         }
+
+        public static void Throws<TNetCoreExceptionType, TNetfxExceptionType>(string paramName, Action action) 
+        where TNetCoreExceptionType : ArgumentException where TNetfxExceptionType : ArgumentException
```

Comment:
Nit: just for symmetry, since the `C` in `Core` is capitalized, consider capitalizing the `F` in `Fx` so that it's `TNetFxExceptionType`